### DISCUSSION TITLE
`mock` module do not install mock/test files

### DISCUSSION THREAD

**Discussion by kloczek** [pypa/setuptools#2751] [General]
I have some difficulties with `mock` package.
I'm trying to package it as rpm and Solaris IPS packages so I'm doing all from non-root account.
Part of every package build procedure is execution of the test suite if it is only possible.
And here is the problem. Looks like setuptools `setup.py build` do not prepares in build/lib mock/tests content and than `setup.py install --root </install/prefix>` is not installing those files as well so by this pytest started with $PYTHONPATH pointing to python site lib/arch directories inside `</install/prefix>` fails because is not able to find `mock.tests` files.

Can someone help a bit and have look on https://github.com/testing-cabal/mock and tell why in this case what is specified in setup.{py,cfg} is not preparing in buil/lib content of the mock/tests?

**Comment by webknjaz** [comment]
Probably because this subpackage is not listed in `setup.cfg`.

**Reply by kloczek** [reply]
> Probably because this subpackage is not listed in `setup.cfg`.

Can you point on some example where something like that is done?

**Reply by webknjaz** [reply]
https://setuptools.rtfd.io/en/latest/userguide/package_discovery.html#package-discovery-and-namespace-package
